server/core/store_option.go

```diff
@@ -82,6 +82,16 @@ func SetStoreState(state metapb.StoreState) StoreCreateOption {
 	}
 }
 
+// OfflineStore offline a store
+func OfflineStore(physicallyDestroyed bool) StoreCreateOption {
```

Comment:
Can we just use `func SetStoreState(state metapb.StoreState, Destroyed ...bool) StoreCreateOption` here?

Reply:
I think `... bool` is not a good design.

Reply:
Or maybe an option?

Reply:
how about
```